sdk/ai/Azure.AI.Agents.Persistent/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Bugs Fixed
 
+- Fixed the data contract for MCP tool.
```

Comment:
Could we add more description on what is the customer facing issue that got fixed?

Reply:
Fixed.